exporters/geneva/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(opentelemetry-fluentd)
```

Comment:
Update the project name?

Reply:
Thanks fixed as `opentelemetry-geneva-metrics`